python/lsst/ts/mtrotator/rotator_csc.py

```diff
@@ -519,13 +548,134 @@ async def do_trackStart(self, data: salobj.BaseMsgType) -> None:
         Once this is run you must issue ``track`` commands at 10-20Hz
         until you are done tracking, then issue the ``stop`` command.
         """
+
+        self.assert_is_motion_locked()
+
         self.assert_enabled_substate(EnabledSubstate.STATIONARY)
         await self.run_command(
             code=enums.CommandCode.SET_ENABLED_SUBSTATE,
             param1=enums.SetEnabledSubstateParam.TRACK,
         )
         self._tracking_started_telemetry_counter = 2
 
+    async def do_lockMotion(self, data: salobj.BaseMsgType) -> None:
+        """Lock the rotator motion.
+
+        Parameters
+        ----------
+        data : `salobj.BaseMsgType`
+            Data of the SAL message.
+
+        Raises
+        ------
+        `salobj.ExpectedError`
+            When the motion is failed to stop.
+        """
+
```

Comment:
I think you also need to add a `self.assert_enabled()` here no?

Reply:
Added in the commit: [b4e39d1](https://github.com/lsst-ts/ts_mtrotator/pull/97/commits/b4e39d11dc643201ed9fc28fd9b89b0c45c65069). Thanks!